subjects/unzipstring/README.md

```diff
@@ -0,0 +1,57 @@
+## Unzipstring
+
+### Instructions
+
+write a function named `Unzipstring` that takes a string in form of a number and printable character that is not a number and returns the original string.
```

Comment:
This readme is really confusing.
Find a way to explain all correctly, and give an example of the expected format.